### PR TITLE
Android filename validation when downloading from browser

### PR DESCRIPTION
**Pull request by jpcloureiro**

### CODE REVIEW

**Comment by jpcloureiro** [comment]
depends on #8 

Merge only after

File: android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt [hunk 1]
```diff
@@ -106,33 +110,46 @@ class RNCWebViewManagerImpl {
...
-            try {
-                urlObj = URL(url)
-                val baseUrl = urlObj.protocol + "://" + urlObj.host
-                val cookie = CookieManager.getInstance().getCookie(baseUrl)
-                request.addRequestHeader("Cookie", cookie)
-            } catch (e: MalformedURLException) {
-                Log.w(TAG, "Error getting cookie for DownloadManager", e)
-            }
+            //Filename validation checking for files that use RTL characters and do not allow those types
+            if (Bidi(fileName, Bidi.DIRECTION_DEFAULT_LEFT_TO_RIGHT).isMixed) {
```
[the remark this answers is not in the document]
Comment (tommasini) [reply android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt:114]: I took a while understanding why `mixed` but yeah it's for catch every filename that use RTL chars like the comment mention!

[the remark this answers is not in the document]
Comment (jpcloureiro) [reply android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt:114]: The code change was taken from [here](https://github.com/MetaMask/metamask-mobile/pull/6844/files#diff-201059220320d45146a2d513851645305e0f908cd746a0ea29a22e2376a6c2d0R257) and improved by Android studio code analyzer

**Review by sethkfman — APPROVED** [review]
LGTM

**Comment by jpcloureiro** [comment]
QA Passed, Wait on #8 to merge

**Pull request merged**